src/backend/legendary/games.ts

```diff
@@ -457,23 +498,29 @@ class LegendaryGame extends Game {
           prefix: LogPrefix.Legendary
         })
       }
-      return { status: 'error' }
+      return { status: 'error', error: res.error }
     }
     return { status: 'done' }
   }
 
   public async uninstall(): Promise<ExecResult> {
     const commandParts = ['uninstall', this.appName, '-y']
 
-    const res = await runLegendaryCommand(commandParts, {
-      logMessagePrefix: `Uninstalling ${this.appName}`
-    })
+    const res = await runLegendaryCommand(
+      commandParts,
+      createAbortController(this.appName),
+      {
+        logMessagePrefix: `Uninstalling ${this.appName}`
+      }
+    )
+
+    deleteAbortController(this.appName)
 
     if (res.error) {
       logError(['Failed to uninstall', `${this.appName}:`, res.error], {
         prefix: LogPrefix.Legendary
       })
-    } else {
+    } else if (!res.abort) {
```

Comment:
Can `res.abort` ever be true here?

Reply:
If aborting the download throws an error this will be true.

Reply:
Aborting via abortController always calls the catch block. We really need to handle aborts. We never did before, if someone aborted the logs where full of errors we should not show.

Reply:
Well, my question was more "Is it possible to abort this specific command". Uninstalling a game should take < 1 second and we're not exposing any buttons/similar in the frontend to cancel the process